lottery/helper-hardhat-config.ts

```diff
@@ -0,0 +1,54 @@
+export interface networkConfigItem {
+    name?: string
+    subscriptionId?: string
+    gasLane?: string
+    keepersUpdateInterval?: string
+    charityRaffleDuration?: string
+    raffleEntranceFee?: string
+    jackpot?: string
+    callbackGasLimit?: string
+    vrfCoordinatorV2?: string
+    charity1?: string
+    charity2?: string
+    charity3?: string
+}
```

Comment:
excellent point, I updated the tests and deployments to stay more true to TS typings and removed mainnet since this is unaudited code for educational purposes